pages/workflow/plan.mdx

```diff
@@ -85,25 +88,22 @@ There are a few things to keep in mind when doing so:
 - Try to apply <Link passHref href="/audit/rules#property-rules"><a>property rules</a></Link> on your properties if possible. This is especially useful when you have a string property that has a finite number of possible values.
 
 <center>
-    <img
-          width="70%"
-          src={require('../../images/workflow/1.plan/event.png')}
-          alt="even definition"
-    />
+  <img
+    width="70%"
+    src={require('../../images/workflow/1.plan/event.png')}
+    alt="even definition"
+  />
 </center>
 
 _Example: Event definition_
 
-
 ## What’s next?
 
-  > <div className={styles.row}>
-          <PageLink
-            image={AnalyticsManagerIcon}
-            title="2. Review suggestions in Avo"
-            description="Learn how to review suggestions in Avo"
-            callToAction={
-              new CallToAction("/workflow/review")
-            }
-          />
-  </div>
+<div className={styles.row}>
+  <PageLink
+    image={AnalyticsManagerIcon}
+    title="2. Review suggestions in Avo"
+    description="Learn how to review suggestions in Avo"
+    callToAction={new CallToAction('https://www.avo.app/docs/workflow/review')}
```

Comment:
Do relative paths not work?

Reply:
Should now after the changes in 078d391!